src/frontend/komponenter/Fagsak/OpprettBehandling/OpprettBehandling.tsx

```diff
@@ -25,36 +28,54 @@ const OpprettBehandling: React.FunctionComponent<IProps> = ({ fagsak }) => {
         settOpprettelseFeilmelding
     );
 
+    const aktivBehandling = hentAktivBehandlingPåFagsak(fagsak);
+    const aktivErÅpen = aktivBehandling
+        ? BehandlingSteg[aktivBehandling.steg] <= BehandlingSteg.BESLUTTE_VEDTAK
+        : false;
+
     return (
         <div className={'opprettbehandling'}>
             <Skjemasteg
                 tittel={'Opprett behandling'}
-                nesteOnClick={() => {
-                    opprettBehandling({
-                        behandlingType: behandlingstype,
-                        søkersIdent: fagsak.søkerFødselsnummer,
-                        kategori: kategori,
-                        underkategori: underkategori,
-                        barnasIdenter:
-                            behandlingstype === Behandlingstype.MIGRERING_FRA_INFOTRYGD
-                                ? barna
-                                      .filter(
-                                          (opprettBehandlingBarn: IOpprettBehandlingBarn) =>
-                                              opprettBehandlingBarn.checked
-                                      )
-                                      .map(
-                                          (opprettBehandlingBarn: IOpprettBehandlingBarn) =>
-                                              opprettBehandlingBarn.barn.personIdent
-                                      )
-                                : [],
-                    });
-                }}
+                nesteOnClick={
+                    !aktivErÅpen
```

Comment:
Dette kunne kanskje vært en egen metode for lesbarhetsskyld?

Reply:
Hvilken del av det? Jeg syntes ofte det er pent med ternary slik at man ikke må gå til egen funksjon for å finne ut hva som skjer.

Reply:
Ah, ser det ble markert feil. Ternaryen må gjerne bli, men fra og med `() => {`

Reply:
Ah, ja - det kan jeg fixe